Makefile

```diff
@@ -13,11 +16,13 @@ clean:
 	rm -rf $(BUILD_DIR)
 
 .PHONY: b2d_iso
-b2d_iso: ISO_NAME=minishift-b2d
+b2d_iso: OSRELEASE_FILE         = os-release
+b2d_iso: OSRELEASE_TEMPLATE     = os-release.template
 b2d_iso: iso_creation
 
 .PHONY: iso_creation
 iso_creation: init
+	isoname='$(ISO_NAME)' version='$(VERSION)' build_id='$(GITTAG)-$(DATE)' envsubst < $(CODE_DIR)/$(OSRELEASE_TEMPLATE) > $(CODE_DIR)/$(OSRELEASE_FILE)
```

Comment:
A comment might be nice here. This is quite hard heavy to take in, or even better move it into its own target, since this is really a dependency to the _iso_creation_ target.